pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1048,6 +1056,16 @@ func doActivate(ctx *domainContext, config types.DomainConfig,
 		if ds.Format != zconfig.Format_CONTAINER {
 			continue
 		}
+
+		snapshotID := getCtrdSnapshotID(ds.FileLocation)
```

Comment:
If there ever is more than one OCI volume for a container you'll do this for each one.
Would that make sense? I guess we would need to mount them all.
This isn't something we support today I hope.